opcua_tools/navigation.py

```diff
@@ -87,6 +87,53 @@ def hierarchical_references_trg_has_no_modelling_rule(
     ].copy()
 
 
+def non_hierarchical_references_trg_has_no_modelling_rule(
+    references: pd.DataFrame, type_references: pd.DataFrame, type_nodes: pd.DataFrame
+) -> pd.DataFrame:
+    """This function expects a table of references on which to find the subset
+    of references containing only references of the subtype "NonHierarchicalReferences" in
+    OPC UA standard, which point to nodes which do no not have a reference to a ModellingRule Node.
+    In order to do this the function also requires tables of the nodes and references which describe
+    the types in the references table.
+
+    This means these are non-hierarchical reference types which do no point

```

Comment:
This argument for non-hierarchical references may be correct, but the actual definition is that it inherits from the reference NonHierarchicalReferences. You could in principle have non-hierarchical references to instancedeclarations also